TF_Tooling_Web/src/tests/TouchFree.test.ts

```diff
@@ -43,4 +44,11 @@ describe('TouchFree', () => {
         expect(ConnectionManager.iPAddress).toBe(newAddress.ip);
         expect(ConnectionManager.port).toBe(newAddress.port);
     });
+
+    it('Should set the cursor correctly', () => {
+        const cursor = new SVGCursor();
+        cursor.DisableCursor();
+        TouchFree.SetCurrentCursor(cursor);
+        expect(TouchFree.GetCurrentCursor()).toBe(cursor);
```

Comment:
For completion sake could we test with DotCursor after this?

Can create one with empty images to make it easier:
`const dotCursor = new DotCursor(new Image(), new Image());`